YoCode/FeatureEvidence.cs

```diff
@@ -29,9 +29,15 @@ public void GiveEvidence(string evidence)
         }
 
         public void SetInconclusive(string reason)
+        {
+            SetInconclusive(new List<string>() { reason });
+        }
+
+        public void SetInconclusive(List<string> reasons)
```

Comment:
Instead of having 2 overloads here you could have one that uses the `params` keyword:
```
public void SetInconclusive(params string[] reasons)
```
That covers both cases and it's probably more "idiomatic" I think.